qutebrowser/utils/urlutils.py

```diff
@@ -349,6 +337,44 @@ def raise_cmdexc_if_invalid(url):
         raise cmdexc.CommandError(get_errstring(url))
 
 
+def get_path_if_valid(pathstr, cwd=None, relative=False, check_exists=False):
```

Comment:
This only seems to be called with `check_exists=True` anyways, so what's the point of having that parameter at all?